src/web/components/ArticleBody.tsx

```diff
@@ -57,9 +66,43 @@ export const ArticleBody = ({
 	adTargeting,
 	host,
 	abTests,
+	pageId,
+	webTitle,
 }: Props) => {
+	if (
+		format.design === Design.LiveBlog ||
+		format.design === Design.DeadBlog
+	) {
+		return (
+			<div
+				className={cx(
+					globalStrongStyles,
+					globalImgStyles,
+					globalLinkStyles(palette),
+				)}
+			>
+				<LiveBlogRenderer
+					format={format}
+					blocks={blocks}
+					adTargeting={adTargeting}
+					host={host}
+					abTests={abTests}
+					pageId={pageId}
+					webTitle={webTitle}
+				/>
+			</div>
+		);
+	}
```

Comment:
I have a feeling we might find this branching to be happening too low down and we might want to do it at the route level. Probably fine for now, but something to be aware of

Reply:
Yeah, I think there's a larger discussion about how we manage rendering liveblogs. Right now though, given how everything thing else currently works, this gets something working on the page

That said... even with a new route, we'd still be asking DCR to render this and there will need to be a branch based on `format` somewhere 🤔